display.py

```diff
@@ -5,14 +5,14 @@
 
 import time
 
-name=raw_input("Enter Your Name : ")
+name=raw_input("Enter Your Name : )
```

Comment:
Ensure the input function is used correctly. It should be `input` instead of `raw_input` in Python 3 to avoid potential issues with data types and security vulnerabilities. Consider updating to `input` for Python 3 compatibility.